Tools/ardupilotwaf/boards.py

```diff
@@ -192,7 +198,7 @@ def configure_env(self, cfg, env):
             'm',
         ]
 
-        cfg.check_librt(env)
+        cfg.check_librt(env, mandatory=False)
```

Comment:
This can't be. What @guludo suggested was to add this to line 146 of the cxx_checks.py file, but even that isn't correct because if we are there it is mandatory to have librt.
Either you change the tested code at the *check_librt* function so that it emulates the real code (dependent on \_\_MACH\_\_ define) or you make an if here checking DEST_OS. @lucasdemarchi I think you didn't like the second option, can you give your input?